torch/testing/_internal/distributed/rpc/dist_optimizer_test.py

```diff
@@ -1,15 +1,17 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
It's odd for a py3 only module to have this line.

Reply:
yep, I think all RPC/dist autograd code was py 3 only, but some parts of the PT codebase are going to support py 2 for a bit longer. I think this should we removed when we officially drop our python 2 support.